### DISCUSSION TITLE
Callout for Blau brand

### DISCUSSION THREAD

**Discussion by DiegodeMiguel** [Telefonica/mistica-design#1677] [Proposals]
### Goal

Accesibility for Blau callout don´t pass actually. (Textsecondary over backgroundContainerAlternative)

### Context

We have received feedback from the OB in Blau about the accesibility for the callout. Actually this fail: 
<img width="1233" alt="image" src="https://github.com/Telefonica/mistica-design/assets/106736588/aaed7c75-3f54-4084-b462-840caab9d630">
I copy here also the link from the tokens https://mistica-design.vercel.app/tokens-map/production/blau/color/backgroundContainerAlternative/
The OB tell us that we need to make it accesible and change the colour, they propose "If you change the grey text to 687173 the ratio is 4.5:1 and works"...but I don´t sure if we need to change the token colour about the "Secondarytext" or it´s better to unify with the other brands and change the background to put it into grey colour, because now Blau its the only one that its different:  
<img width="585" alt="image" src="https://github.com/Telefonica/mistica-design/assets/106736588/27de2570-4723-484d-853b-6cec38278390">
Thanks.

### Impact

This affect to all callouts. I attach a figma link as example https://www.figma.com/file/NPyjxLokVg6KLkdVrvkR7w/Blau_INVOICES?type=design&node-id=3958-14638&mode=design&t=pQAiNC4CAxvmybte-4

### Discussion Questions

Should we evolve this component or just find a new token colour?

**Comment by aweell** [comment]
Hey @DiegodeMiguel,

We can create an issue to modify `text-secondary` from `#808285` to `#687173` in Blau, be aware that this change will affect transversally so other elements will change.

Regarding the callout background, @HectorSancho already raise the possibility to soften that color.

Since there can be more concerns in the future about the a11y of color usage in the Blau skin, I suggest a more holistic approach in order to identify the problems that can be solved on our side instead of going one by one since there are inter-dependences between multiple color tokens in the system that may be difficult to foresee.

**Comment by aweell** [comment]
Buenas @DiegodeMiguel, revisando la documentación de Blau en brand Factory y la [definición de color en One design system](https://one.o9.de/main/blau/docs/index.html?path=/story/good-to-know-design-tokens--page), hemos encontrado el neutral700 cuyo valor es `#666666`, por mantener la consistencia entre las diferentes implementaciones de la marca Blau nuestra idea era utilizar este color.
